docs/site/tutorials/custom_differentiation.ipynb

```diff
@@ -469,7 +469,7 @@
    "source": [
     "## Recomputing activations during backpropagation to save memory (checkpointing)\n",
     "\n",
-    "Checkpointing is a traditional technique in reverse-mode automatic differentiation to save memory when computing derivatives by making large intermediate values in the original computation not be saved in memory for backpropagation, but instead recomputed as needed during backpropagation. This technique has been realized in modern deep learning libraries as well. In Swift, API [`withRecomputationInPullbacks(_:)`](https://www.tensorflow.org/swift/api_docs/Protocols/Differentiable#/s:10TensorFlow14DifferentiablePAAE28withRecomputationInPullbacksyqd__qd__xcAaBRd__lF) makes you able to control what to recompute during backpropagation, and it is available on all `Differentiable` types.\n",
+    "Checkpointing is a traditional technique in reverse-mode automatic differentiation to save memory when computing derivatives by making large intermediate values in the original computation not be saved in memory for backpropagation, but instead recomputed as needed during backpropagation. This technique has been realized in modern deep learning libraries as well. In Swift, API [`withRecomputationInPullbacks(_:)`](https://www.tensorflow.org/swift/api_docs/Protocols/Differentiable#/s:10TensorFlow14DifferentiablePAAE28withRecomputationInPullbacksyqd__qd__xcAaBRd__lF) enables you to control what to recompute during backpropagation, and it is available on all `Differentiable` types.\n",
```

Comment:
I might suggesting breaking up the first sentence:
```
Checkpointing is a traditional technique in reverse-mode automatic differentiation for saving memory.
Rather than saving large intermediate values in the original computation for computing derivatives,
the intermediate values are instead recomputed as needed during backpropagation.
```

Reply:
Good points, thanks. Fixed.